qiita_db/handlers/artifact.py

```diff
@@ -212,7 +207,9 @@ def post(self):
         try:
             qdb.artifact.Artifact.create_type(a_type, a_desc, ebi, vamps,
                                               fp_types)
-        except qdb.exceptions.QiitaDBDuplicateError as e:
-            raise HTTPError(400, str(e))
+        except qdb.exceptions.QiitaDBDuplicateError:
+            # Ignoring this error as we want this endpoint in the rest api
+            # to be idempotent.
+            pass
```

Comment:
I think you can also set the `_reason` to the response so that there's a difference between it already exists and everything went fine. Not blocking though. http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.set_status